src/utils/Card.js

```diff
@@ -250,8 +250,13 @@ export const cardElo = (card) => card.details.elo;
 
 export const cardLayout = (card) => card.details.layout;
 
-export const cardDevotion = (card, color) =>
-  cardCost(card)?.reduce((count, symbol) => count + (symbol.includes(color.toLowerCase()) ? 1 : 0), 0) ?? 0;
+export const cardDevotion = (card, color) => {
+  console.log(card.details.name);
+  console.log(card);
+  let cost = cardCost(card);
+  if (cost && cardLayout(card) === 'adventure') cost = cost.slice(cost.findIndex((x) => x === 'split') + 1);
```

Comment:
Mana costs are reversed when parsing, so the cost of the permanent half is actually *after* the `'split'` symbol.

The layout check needs to be here because split cards also have a `'split'` mana symbol, and for them it makes more sense to count both halves.